BLSQ/iaso
# IA-2056: registry part 3

Multiple improvements on registry 

Related JIRA tickets : IA-2111, IA-2108, IA-2107, IA-2105, IA-2106

## Self proofreading checklist


## Changes

- Review of navigation using replace for `LinkForRegistry` (refactor of `LinkToOrgUnit`)
- Allow to toggle tooltip for children org units
- Review preselected sub org unit type logic
- display count of sub org units tin tabs
- link to submissions everywhere

## How to test

Go to registry page for an org unit, play with the navigation and come back, check that all required features are working


## Print screen / video

https://user-images.githubusercontent.com/12494624/234879571-e2f50a32-79a8-4e2f-abbe-cc5904ee6069.mov
